File: Python/citoplasma.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import celula
from typing import List

class citoplasma(object):
	def getCantidadSal(self) -> int:
		return self.___cantidadSal

	def setCantidadSal(self, aCantidadSal : int) -> None:
		self.___cantidadSal = aCantidadSal

	def getCantidadAgua(self) -> int:
		return self.___cantidadAgua

	def setCantidadAgua(self, aCantidadAgua : int) -> None:
		self.___cantidadAgua = aCantidadAgua

	def __init__(self):
		self.___cantidadSal : int = None
		self.___cantidadAgua : int = None
		self._unnamed_celula_ : celula = None
		"""# @AssociationKind Composition"""

File: C#/citoplasma.cs
using System;

public class Citoplasma {
	private int cantidadSal;
	private int cantidadAgua;

	public int GetCantidadSal() {
		return this.cantidadSal;
	}
	public void SetCantidadSal(ref int cantidadSal) {
		this.cantidadSal = cantidadSal;
	}
	public int GetCantidadAgua() {
		return this.cantidadAgua;
	}
	public void SetCantidadAgua(ref int cantidadAgua) {
		this.cantidadAgua = cantidadAgua;
	}

	private Celula celula;

}
